languages/js/src/Host.ts

```diff
@@ -359,9 +359,15 @@ export class Host implements Required<DataFilteringQueryParams> {
    *
    * @internal
    */
-  isaWithPath(baseTag: string, path: string[], classTag: string): boolean {
+  async isaWithPath(
+    baseTag: string,
+    path: PolarTerm[],
+    classTag: string
+  ): Promise<boolean> {
     let tag = baseTag;
-    for (const field of path) {
+    for (const fld of path) {
+      const field = await this.toJs(fld);
+      if (!isString(field)) throw new Error();
```

Comment:
Should provide an error message here.

Reply:
Wouldn't hitting this error mean we'd broken the `isa_with_path` API in the core? I.e., I don't think users should ever see this error. I guess it isn't a bad idea to add a message for ourselves; just not something we do w/ much (any?) of our other validation code in the JS lib that's validating data that we're in control of.

Reply:
@gj that's correct. I'm indifferent 🤷‍♀️